content/en/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use.md

```diff
@@ -93,6 +93,7 @@ nodes.
         name to find out which runtime you use. For example,
         `unix:///run/containerd/containerd.sock` is the containerd endpoint.
 
-If you use Docker Engine with the dockershim, [migrate to a different runtime](/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd/),
-or, if you want to continue using Docker Engine in v1.24 and later, migrate to a
+If you want to change the Container Runtime on a Node from Docker Engine to containerd,
+you can find out more information on [migrate Docker Engine to  containerd](/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd/),
```

Comment:
I'd write:
```suggestion
you can find out more information on [migrating from Docker Engine to containerd](/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd/),
```

Reply:
@sftim  thank you for your suggestion.  Sorry for the delayed  commit. 😄